src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -718,13 +718,18 @@ protected static function getBundleDefaultConfig()
                 'x-forwarded-proto',
             ],
             'csrf_protection' => [
-                'enabled' => false,
+                'enabled' => null,
+                'cookie_name' => 'x-csrf-token',
```

Comment:
Using `x-` prefix is not recommended anymore: https://www.rfc-editor.org/rfc/rfc6648. `symfony-csrf-token`? OWASP used `anti-csrf-token` for the header.

Reply:
`symfony-` may something people want to hide.

Reply:
Good call, I'm going with just `csrf-token`.